pkg/sidecar/config.go

```diff
@@ -30,6 +30,11 @@ type Config struct {
 	SvidBundleFileNameDeprecated       string `hcl:"svidBundleFileName"`
 	RenewSignal                        string `hcl:"renew_signal"`
 	RenewSignalDeprecated              string `hcl:"renewSignal"`
+
+	// JWT configuration
+	JwtAudience  string `hcl:"audience"`
+	JSONFilename string `hcl:"json_filename"`
```

Comment:
Do you think it'd be clearer to name this `jwt_filename`?

Reply:
Yes, the only issue I see is that the JSON could also contains JWKs, so it's not only a JWT

Reply:
@keeganwitt @faisal-memon I just updated JSONFilename, separated it into two different files:
* jwt_file_name : contains the JWT SVID.
* jwk_file_name : contains the JWT Bundle.

Reply:
Nice! That's much more useful.